docs/2/technical/data-format/daily-snapshot.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: docs
+title: Daily Snapshot Data Format
+description: Let's take a look at the Microshare™ Daily Snapshot data format
+toc: true
+---
+
+{% include image.html url="/assets/img/thumbnail-8.jpg" height="900" width="900" description="thumbnail 2" %}
+
+
+<br>
+
+---------------------------------------
+
+##### SUMMARY: 
+
+1. [Overview](./#1-overview)
+2. [Daily snapshot data format](./#2-daily-snapshot-data-format)
+
+---------------------------------------
+
+## 1. Overview
+---------------------------------------
+This page provides an overview of the data structure for the daily snapshot data format used in the [predictive cleaning](https://www.microshare.io/eversmart-predictive-cleaning/) solution developed by Microshare™. The data is presented in JSON format and contains daily aggregated snapshots of a given metric produced by a device.
+
+
```

Comment:
Can you add a paragraph about the rectypes for each sections so people can query it. And also maybe put an eplanation that the metric/category can help querying the hourly snapshot / daily snapshot for different data types. 
And for the hourly snapshot can you explain the meaning of an hourly aggregation to offload the quantity of data and be able to query a lot more analytics.

Reply:
I've made the changes. Please check when you get the time.